### DISCUSSION TITLE
R2. Consistent gebruik van objecttypering voor soorten objecten

### DISCUSSION THREAD

**Discussion by RiX012** [Geonovum/disgeo-imsor#45] [General] [answered]
Dit proeft voor mij als 'single inheretence'. Het lijkt me niet verstandig om dat als uitgangspunt te nemen. Misschien bij dit uitgangspunt expliciet vermelden of dit wel of niet het geval is.

**Comment by pmaria** [comment]
> Dit proeft voor mij als 'single inheretence'. 

Dat klopt grotendeels, single inheritance en single typing. 

>  Het lijkt me niet verstandig om dat als uitgangspunt te nemen. Misschien bij dit uitgangspunt expliciet vermelden of dit wel of niet het geval is.

Ik ben het met je eens dat je dit in een puur semantisch model niet zou willen, echter in een logisch informatiemodel kun je vaak niet anders. De meeste afnemers gebruiken informatie in software, en de meeste programmeertalen hanteren (jammer genoeg) een single typed perspectief. Omdat we willen dat het logische informatiemodel breed toepasbaar is hebben we deze richtlijn opgenomen.

**Reply by RiX012** [reply]
Ok. Goeie overweging. Dank

**Reply by MichelBohms** [reply]
ld model is toch logisch model > dat wel behoefte aan multiple I/T

**Comment by pmaria** [comment, marked as answer]
Voorlopig hebben we deze richtlijn geschrapt. We willen nog geen beperkingen op meervoudig overerven opleggen. We gaan dit wel verder onderzoeken.
